extensions/color-picker/src/color-names.tsx

```diff
@@ -17,6 +17,7 @@ export default function ColorNames() {
     setIsSearching(true);
     try {
       const colors = colorNamer(searchString);
+      console.log(colors);
```

Comment:
Does this console log need to be reserved?

Reply:
Good catch!